update-dependencies/CliDependencyHelper.cs

```diff
@@ -38,11 +38,11 @@ public string GetSharedFrameworkVersion()
             XNamespace msbuildNamespace = depVersions.Document.Root.GetDefaultNamespace();
             string sharedFrameworkVersion = depVersions.Document.Root
                 .Element(msbuildNamespace + "PropertyGroup")
-                ?.Element(msbuildNamespace + "CLI_SharedFrameworkVersion")
+                ?.Element(msbuildNamespace + "MicrosoftNETCoreAppPackageVersion")
```

Comment:
Suggest declaring this as a const, and using that here and in the throw message.

Reply:
fixed